src/Sulu/Bundle/ContentBundle/Automation/DocumentPublishHandler.php

```diff
@@ -11,64 +11,28 @@
 
 namespace Sulu\Bundle\ContentBundle\Automation;
 
-use Sulu\Bundle\AutomationBundle\TaskHandler\AutomationTaskHandlerInterface;
-use Sulu\Bundle\AutomationBundle\TaskHandler\TaskHandlerConfiguration;
 use Sulu\Component\Content\Document\Behavior\WorkflowStageBehavior;
-use Sulu\Component\DocumentManager\DocumentManagerInterface;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
  * Provides handler for publishing documents.
  */
-class DocumentPublishHandler implements AutomationTaskHandlerInterface
+class DocumentPublishHandler extends BaseDocumentHandler
 {
     /**
-     * @var DocumentManagerInterface
+     * @var string
      */
-    private $documentManager;
+    protected $title = 'sulu_content.task_handler.publish';
```

Comment:
i like it basically this way because it lasts into smaller constructors. but i will restructure it.